migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,394 @@
+
+== Procedure
+
+. Navigate to the {PRODUCT_DOWNLOAD_PAGE}[{PRODUCT_LONG} v{VERSION_OLD} Downloads Page] and follow the instructions to download the {PRODUCT} **Update Tool**.  
+
++ 
+If you are upgrading to a new minor release of {PRODUCT}, first apply the latest patch update to your current version of {PRODUCT} and then follow this procedure again to upgrade to the new minor release.
+
+. Extract the downloaded file to a temporary directory.
+
++
+This update tool automates the update of certain components of {PRODUCT}, such as Decision/Business Central, KIE Server, and the headless controller. Use this update tool first to apply updates and then install any other updates or new release artifacts that are relevant to your {PRODUCT} distribution.
+
+. If you want to preserve any files from being updated by the update tool, navigate to the extracted `bamoe-$VERSION-update` folder, open the `blacklist.txt` file, and add the relative paths to the files that you do not want to be updated.
+
++
+When a file is listed in the `blacklist.txt` file, the update script does not replace the file with the new version but instead leaves the file in place and in the same location adds the new version with a `.new` suffix. If you block files that are no longer being distributed, the update tool creates an empty marker file with a `.removed` suffix. You can then choose to retain, merge, or delete these new files manually.
+
++ 
+Example files to be excluded in `blacklist.txt` file:
+
++ 
+[source,shell,subs="attributes+"]
+----
+WEB-INF/web.xml  // Custom file
+styles/base.css  // Obsolete custom file kept for record
+----
+
++
+The contents of the blocked file directories after the update:
+
++
+[source,shell,subs="attributes+"]
+----
+$ ls WEB-INF
+web.xml web.xml.new
+
+$ ls styles
+base.css base.css.removed
+----
+
+. In your command terminal, navigate to the temporary directory where you extracted the `bamoe-$VERSION-update.zip` file and run the _apply-updates_ script in the following format:
+
++
+[IMPORTANT]
+====
+Make sure that your Decision/Business Central and KIE Server instances are not running before you apply updates. Do not apply updates while you are running an instance of Decision/Business Central or KIE Server.
+====
+
++
+On Linux or Unix-based systems:
+
++
+[source,shell,subs="attributes+"]
+----
+$ ./apply-updates.sh $DISTRO_PATH $DISTRO_TYPE
+----
+
++
+On Windows:
+
++
+[source,shell,subs="attributes+"]
+----
+$ .\apply-updates.bat $DISTRO_PATH $DISTRO_TYPE
+----
+
++
+The `$DISTRO_PATH` portion is the path to the relevant distribution directory and the `$DISTRO_TYPE` portion is the type of distribution that you are updating with this update.  The following distribution types are supported in {PRODUCT} update tool:
+
++
+
+.Distribution Types for {PRODUCT} v{VERSION_OLD}
+|===
+| Archive | Description
+
+| bamoe-8.0-BC7   
+| Updates Business Central (`business-central.war`)`
+
+| IBAMOE-8.0-KS8
```

Comment:
is `IBAMOE` prefix correct here? just asking do not know, other in the table have prefix `bamoe`

Reply:
Good question. @blulas @porcelli ?

Reply:
Alex said IBAMOE is not correct. I'll get this fixed.

Reply:
@YajieY to help!!! 

@YajieY could you help with this file names? :)

Reply:
fixed